src/applications/mhv/secure-messaging/components/FoldersList.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { folderPathByFolderId } from '../util/helpers';
-import { DefaultFolders as Folders } from '../util/constants';
+import { folderPathByFolderId, updatePageTitle } from '../util/helpers';
+import { DefaultFolders as Folders, PageTitles } from '../util/constants';
 
 const FoldersList = props => {
   const { folders, showUnread } = props;
 
+  const handleFolderPageTitles = newTitle => {
+    updatePageTitle(`${newTitle} ${PageTitles.PAGE_TITLE_TAG}`);
+  };
+

```

Comment:
I am curious if this is necessary if you are already handling this in `Folders.jsx`